### DISCUSSION TITLE
Problem regarding Battle #1 Target #9

### DISCUSSION THREAD

**Discussion by sandesh1010** [cssbattle/forum#78] [General]
![image](https://user-images.githubusercontent.com/84668253/124354123-20aedd00-dc2a-11eb-889c-e4ded1e160cc.png)

I encountered a problem in cssbattle with this code. The output is visually same but I get only 70.8% match when I submit it. Can anyone help me on this problem?

``` 
<div a></div>
<div b></div>
<div c></div>
<style>
body{
    background:#222730;
    display:grid;
    place-items:center;
    height:100%
}
[a]{
    width:50;
    height:50;
    background:#393E46;
    position:absolute;
    border-radius:50%;
    top:125;
    z-index:1;}
[b]{
    width:150;
    height:150;
    transform:rotate(45deg);
    background:#4CAAB3; 
    box-shadow:0 0 0 50px #222730;
    top:75;
    position:absolute;
}
[c]{
    height:150;
    width:400;
    background:#4CAAB3;
    position:absolute;
    z-index:-1;
    top:75;
}
</style>

```

**Comment by RedHillE** [comment]
Hi,
your problem is you are using Firefox.  This is what it looks like in Chrome :
![chrome](https://user-images.githubusercontent.com/87064609/124727740-e33aa000-df06-11eb-907e-18a1285a386f.png)
You can see the percentage is the same but it's way off appearance wise.
Chrome is the recommended browser to do battle in
